app/models/hearings/forms/base_hearing_update_form.rb

```diff
@@ -208,30 +205,99 @@ def virtual_hearing_updates
       representative_email_sent: representative_email_sent_flag
     }.reject { |_k, email_sent| email_sent == true }
 
-    sanitize_updated_emails if virtual_hearing_attributes.present?
-
     updates = (virtual_hearing_attributes || {}).merge(emails_sent_updates)
 
     if judge_id.present?
-      updates[:judge_email] = hearing.judge&.email
+      updates[:judge_email] = judge_email
     end
 
     updates
   end
 
+  def virtual_hearing_updates
+    @virtual_hearing_updates ||= define_virtual_hearing_updates
+  end
+
   def virtual_hearing_created?
     @virtual_hearing_created ||= false
   end
 
+  def update_appellant_recipient
+    if appellant_email.present?
+      hearing.appellant_recipient.update!(email_address: appellant_email)
+    end
+
+    if virtual_hearing_updates[:appellant_tz].present?
+      hearing.appellant_recipient.update!( timezone: virtual_hearing_updates[:appellant_tz])
+    end
+
+    if virtual_hearing_updates.key?(:appellant_email_sent)
+      hearing.appellant_recipient.update!(email_sent: virtual_hearing_updates[:appellant_email_sent])
+    end
+  end
```

Comment:
```suggestion
  def update_appellant_recipient
    update_params = {
      email_address: appellant_email.presence,
      timezone: virtual_hearing_updates[:appellant_tz].presence,
      email_sent: virtual_hearing_updates[:appellant_email_sent]
    }.compact

    hearing.appellant_recipient.update!(**update_params) if update_params.any?
  end
```
I think this would give you the same result and be more efficient (0-1 updates rather than 0-3)